iroha/src/block_sync.rs

```diff
@@ -11,7 +11,9 @@ use self::{
 };
 use crate::{
     prelude::*,
-    sumeragi::{CommitBlock, GetNetworkTopology, GetSortedPeers, Role, SumeragiTrait},
+    sumeragi::{
+        network_topology::Role, CommitBlock, GetNetworkTopology, GetSortedPeers, SumeragiTrait,
```

Comment:
Btw, should we remove substitue network topology with just topology? Topology 90% of times is about network.

Reply:
I think it is better to leave module name like this, but yeah I changed the struct name

Reply:
I mean you won't mix it up with anything. So it seems that there are not much of reasons for that. Especially in the context of sumeragi.

Reply:
Still, I think it adds clarity. I don't think it is worth arguing about it too much.